src/VectorTypes.f90

```diff
@@ -1563,6 +414,10 @@ SUBROUTINE axpy_scalar_VectorType(thisVector,newVector,a,n,incx,incy)
 !> @param incx the increment to use when looping over elements in @c x
 !> @param incy the increment to use when looping over elements in @c y
 !>
+!> MAINTENANCE NOTE: This is inefficient. It makes a copy of the vectors, which
```

Comment:
I used MAINTENANCE NOTE before using TIBWSFB. If you think its a big enough deal, I'll make an issue.